### ISSUE TITLE
Static js/css files. ( Remove redirects )

### ISSUE THREAD

**Issue by realrandal** [gobezu/jbetolo#28]
As google suggest:

> Minimizing HTTP redirects from one URL to another cuts out additional RTTs and wait time for users.

What's google says it's not so important. But true reason is that: For example, static compression from Warp6/Widgetkit and others, is directly to cached files. Whole content of css is inside linked files.

In real world, For example: Google Chrome FLASH white page on during pages switch. 
I think the effect is caused by redirections. Files are not serving directly inside CSS or JS files. All it destroy feeling like page is FAST even if page is highly simple and optimized.

When disable jbetolo, left page without any merge and compression. Raw files load at first longer. But when you navigate on site you don't feel it's weight. Because everything is cached and browser don't download them again. So pages can switch without even 1ms of white page feeling. 

So BIG ASK is for option caching files without redirects. Without it all jbetolo job are a little bit pointless for me. 
On http://jproven.com site you use CDN and this effect does not appear. But we talk about regular site without CDN.

In other words get rid off reflows like:

> index.php?option=com_jbetolo&task=serve&cfile=1_970321:34:0

Thank you.

**Comment by Sophist-UK** [comment]
I note that we get redirects on combined files if we have .htaccess functionality set in jbetolo. These should not show white flashes, however there does not appear to be a real need for redirects like this rather than loading the files direct.

Note: The htaccess functionality is designed to provide a query-less URL i.e. /cache/hash.css rather than index.php?stuff in order to allow the file to used from the cache, however if it only triggers a 301/302 redirect to the query url, then it will not have this effect.
